src/main/java/erb/ui/RouteAddDialog.java

```diff
@@ -209,7 +200,8 @@ public void actionPerformed(ActionEvent e) {
     //ok button
     void okB_actionPerformed(ActionEvent e) {
     	//erb: vk US-57-TASK-106 only adding route name and length and duration will be calculated
-    	RouteManager.createRoute(addRouteNameField.getText());
+        //erb: vk TODO US-57-117 will need to get the path from US57-Task109
+    	RouteManager.createRoute(addRouteNameField.getText(), "1,2,5");
```

Comment:
change to reflect a real path to demo end game functionality
```suggestion
    	RouteManager.createRoute(addRouteNameField.getText(), "1,2,60,6,7,10,11,12,9,8,7,6,5,4,3,1");
```

Reply:
Updated this.  :D